.do/deploy.template.yaml

```
@@ -6,16 +6,34 @@ spec:
         - key: SMTP_HOST
         - key: SMTP_PORT
         - key: SMTP_USER
         - key: SMTP_PASSWORD
         - key: SMTP_FROM
-        - key: DATABASE_URL
         - key: SVIX_URL
         - key: SVIX_API_KEY
+      http_port: 4002
+      instance_count: 1
+      instance_size_slug: basic-xxs
+      name: saas-starter-kit
```

YAMLlint (1.37.1)

[error] duplication of key "name" in mapping

(key-duplicates)